app/views/posts/index.html.erb

```diff
@@ -1 +1,33 @@
-<h1>Here is a list of all posts by user 1</h1>
\ No newline at end of file
+<article class="main-article" style="width: 90%; margin: auto">
+  <div style="display: flex; align-items: center; justify-content: space-between;">
+    <%= image_tag @user.photo, :class => "user-avatar" %>
+    <div class="user-details">
+      <p><%= @user.name %></p>
+      <span>Number of posts: <%= @user.post_counter %></span>
+    </div>
+  </div>
+
+  <% @user.recent_posts.each_with_index do |post, index| %>
+    <div>
+      <div class="post">
+        <h4>Post #<%= index + 1 %></h4>
```

Comment:
- [ ] [Optional] Well! good job so far. Here, you are showing a placeholder where the index is dynamically changing from `Post #1`, `Post #2`, and so on. However, in the wireframe, it shows the project title as `Post#1`, `Post#2`. I know the wireframe indicates the same as you have, but it's a suggestion to consider changing the placeholder to match the project title for a more user-friendly display.

Reply:
@shahadat3669 The Wireframe seems to render the Post placeholder dynamically. I can't understand your point, please elaborate.

Reply:
@simplegoose, after a thorough review of the wireframes, it's apparent that it's presenting project-specific information without relying on generic placeholders. Each wireframe is carefully crafted to include functional elements such as user details (names, post counts) and post-specific details (title, likes count), all seamlessly pulled from the database. Notably, even the comments are sourced directly from the database, enhancing the overall authenticity of the user experience. 🛠️📊

Considering this approach, where every piece of data comes dynamically from the database, the question arises: why contemplate using placeholders for posts? ❓

When we look closely at the wireframes, it's clear that the creator added a personal touch by intentionally giving the title 'post #1.' This shows that careful and thoughtful design choices were made during the wireframing process. 🎨✨

Additionally, this is an optional change; you can update your code according to the comment or you can ignore it. But I will recommend you to update it. 🔄💡

**_Note: I apologize for my late reply. I took a break and did not see your comments. Thank you.🙏_**